lib/sql.js

```diff
@@ -680,6 +680,24 @@ SQLConnector.prototype.update = function(model, where, data, options, cb) {
 };
 
 /**
+ * Update all instances that match the where clause with the given data, but
+ * without returning the count.
+ * @param {String} model The model name
+ * @param {Object} where The where object
+ * @param {Object} data The property/value object representing changes
+ * to be made
+ * @param {Object} options The options object
+ * @param {Function} cb The callback function
+ */
+SQLConnector.prototype.updateNoCount = function(model, where, data,
+  options, cb) {
+  var stmt = this.buildUpdate(model, where, data, options);
+  this.execute(stmt.sql, stmt.params, options, function(err) {
```

Comment:
Noticed in the other PR we are wrapping the statement now. So this is not an issue.